internal/services/keyvault/key_vault_certificate_data_source.go

```diff
@@ -39,6 +39,16 @@ func dataSourceKeyVaultCertificate() *pluginsdk.Resource {
 				ValidateFunc: keyVaultValidate.VaultID,
 			},
 
+			"resource_id": {
+				Computed: true,
+				Type:     pluginsdk.TypeString,
+			},
+
+			"resource_versionless_id": {
+				Computed: true,
+				Type:     pluginsdk.TypeString,
+			},
```

Comment:
We should be more explicit in what kind of ID we're referring to so can we rename these to

```suggestion
			"resource_manager_id": {
				Computed: true,
				Type:     pluginsdk.TypeString,
			},

			"resource_manager_versionless_id": {
				Computed: true,
				Type:     pluginsdk.TypeString,
			},
```

I'm aware we have fields called `resource_id` and `resource_versionless_id` in some other key vault resources - these will need to be renamed to `resource_manager_id` and `resource_manager_versionless_id` for consistency in 4.0

Reply:
thanks for your information. i have renamed these fileds.